source/index.html

```diff
@@ -1,12 +1,193 @@
+              <div class="companion-card__level-bar level-bar"><span class="level-bar__level">80</span><span class="level-bar__text">level</span></div>
+              <ul class="companion-card__destinations destinations-list">
+                <li class="destinations-list__item"><img src="https://via.placeholder.com/35x25" alt="Бельгия" width="35" height="25"></li>
+                <li class="destinations-list__item"><img src="https://via.placeholder.com/35x25" alt="Чехия" width="35" height="25"></li>
+              </ul>
+              <h4 class="companion-card__name">Петя Демин</h4>
+              <ul class="companion-card__hashtags hashtags-list">
+                <li class="hashtags-list__item">#бургер</li>
+                <li class="hashtags-list__item">#бар</li>
+                <li class="hashtags-list__item">#футбол</li>
+                <li class="hashtags-list__item">#концерт</li>
+                <li class="hashtags-list__item">#крафт</li>
+              </ul>
+              <ul class="companion-card__transport transport-list">
+                <li class="transport-list__item" aria-label="Авиаперелет"></li>
+                <li class="transport-list__item" aria-label="Автотранспорт"></li>
+                <li class="transport-list__item" aria-label="Велосипед"></li>
+                <li class="transport-list__item" aria-label="Пешком"></li>
+              </ul>
+            </article>
+          </li>
+        </ul>
+        <a href="./catalog.html" class="companions__button button">Показать всех</a>
+      </section>
+      <section class="rates">
+        <h3 class="rates__heading">Добавьте свой профиль</h3>
+          <ol class="rates__list rates-list">
+            <li class="rates-list__item">
+              <p class="rates-list__service">Базовое размещение</p>
+              <p class="rates-list__price">Бесплатно</p>
+            </li>
+            <li>
+              <p class="rates-list__service">Вывод на первых позициях</p>
+              <p class="rates-list__price">99 руб./мес</p>
+            </li>
+            <li>
+              <p class="rates-list__service">Кастомный цвет карточки</p>
+              <p class="rates-list__price">59 руб./мес</p>
+            </li>
+          </ol>
+        <a href="rates.html" class="rates__link">Показать тарифы для бизнеса</a>
+      </section>
+      <section class="login">
+        <h3 class="login__heading">Заинтересовались?</h3>
+        <p class="login__text">Зарегистрируйтесь, это быстро и бесплатно!</p>
+        <p class="login__text">Уже зарегистрированы? Тогда скорее <a class="login__login" href="login.html">авторизуйтесь</a>.</p>
+        <form action="https://echo.htmlacademy.ru" method="post" class="login__form">
+          <label class="visually-hidden" for="email-field">E-mail</label>
+          <input class="login__email" type="email" placeholder="E-mail" id="email-field">
+          <button class="login__button button button--mini"></button>
```

Comment:
`type` внутри формы обязателен, так как не все кнопки могут быть для отправки формы
```suggestion
          <button class="login__button button button--mini" type="submit"></button>
```